code/modules/mining/equipment/kinetic_crusher/trophies/lavaland_mob_trophies.dm

```diff
@@ -0,0 +1,208 @@
+	return "mark detonation to prevent certain creatures from using certain attacks for <b>[bonus_value * 0.1]</b> second\s"
+
+/obj/item/crusher_trophy/watcher_wing/on_mark_detonation(mob/living/target, mob/living/user)
+	if(ishostile(target))
+		var/mob/living/simple_animal/hostile/victim = target //this is fun, doesn't apply to basic mobs. need to look into that
+		if(victim.ranged) //briefly delay ranged attacks
+			if(victim.ranged_cooldown >= world.time)
+				victim.ranged_cooldown += bonus_value
+			else
+				victim.ranged_cooldown = bonus_value + world.time
+
+/**
+ * Watcher - icewing variant
+ * Detonating a mark causes the victim's next ranged attack/special attack to cool down even slower.
+ */
+/obj/item/crusher_trophy/watcher_wing/ice_wing
+	name = "icewing watcher wing"
+	desc = "A carefully preserved frozen wing from an icewing watcher. Suitable as a trophy for a kinetic crusher."
+	icon_state = "ice_wing"
+	bonus_value = 8
+
+/**
+ * Watcher - magmawing variant
+ * Detonating a mark causes the next destabilizer shot to deal damage.
+ * The bonus lasts for a single shot and resets after 30 seconds.
+ */
+/obj/item/crusher_trophy/blaster_tubes/magma_wing
+	name = "magmawing watcher wing"
+	desc = "A still-searing wing from a magmawing watcher. Suitable as a trophy for a kinetic crusher."
+	icon_state = "magma_wing"
+	gender = NEUTER
+	bonus_value = 5
+
+/obj/item/crusher_trophy/blaster_tubes/magma_wing/effect_desc()
+	return "mark detonation to make the next destabilizer shot deal <b>[bonus_value]</b> damage"
+
+/obj/item/crusher_trophy/blaster_tubes/magma_wing/on_projectile_fire(obj/projectile/destabilizer/marker, mob/living/user)
+	if(deadly_shot)
+		marker.name = "heated [marker.name]"
+		marker.icon_state = "lava"
+		marker.damage = bonus_value
+		marker.AddComponent(/datum/component/crusher_damage_ticker, APPLY_WITH_PROJECTILE, bonus_value)
+		deadly_shot = FALSE
+
+/**
+ * Legion - the carbon-sized one
+ * Causes the crusher's destabilizer shot to recharge faster.
+ */
+/obj/item/crusher_trophy/legion_skull
+	name = "legion skull"
+	desc = "A dead and lifeless legion skull. Suitable as a trophy for a kinetic crusher."
+	icon_state = "legion_skull"
+	denied_type = /obj/item/crusher_trophy/legion_skull
+	bonus_value = 3
+
+/obj/item/crusher_trophy/legion_skull/effect_desc()
+	return "a kinetic crusher to recharge <b>[bonus_value*0.1]</b> seconds faster"
+
+/obj/item/crusher_trophy/legion_skull/add_to(obj/item/kinetic_crusher/target_crusher, mob/living/user)
+	. = ..()
+	if(.)
+		target_crusher.charge_time -= bonus_value
+
+/obj/item/crusher_trophy/legion_skull/remove_from(obj/item/kinetic_crusher/target_crusher, mob/living/user)
+	. = ..()
+	if(.)
+		target_crusher.charge_time += bonus_value
+
+/**
+ * Lobstrosity
+ * Detonating a mark causes the victim to stagger for 1 second, which slows down movement
+ * and more than doubles the cooldown for the victim's ranged attack (if it's a simple hostile mob).
```

Comment:
...so it largely also doesn't do anything 😔 
making these work doesnt necessarily need to be in this PR but it's sad